packages/app/src/systems/Chains/eth/hooks/useEthAccountConnection.tsx

```diff
@@ -21,7 +22,10 @@ export function useEthAccountConnection(props?: {
   const { address, isConnected } = useAccount();
   const { data: ensName } = useEnsName({ address });
   const { data: ensAvatar } = useEnsAvatar({ name: address });
-  const { data: balance } = useBalance({ address, token: erc20Address });
+  const { data: balance } = useBalance({
+    address,
+    token: erc20Address !== NativeAssetId ? erc20Address : undefined,
```

Comment:
this will never happen right`erc20Address !== NativeAssetId`? now that eth is `undefined` in eth side